<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plotter_scatter3d</class>
 <widget class="QWidget" name="plotter_scatter3d">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="gbxPoints">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Point cloud</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="loForPointClouds"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbxPointCloudControls">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Point cloud settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lblHint">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;1. Use scroll to zoom, use mouse with holding right button to orientate camera.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="blbFormatOfPoint">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>2. Click on point to display coordinates in format [XYZ]</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="lblShadowQuality">
          <property name="text">
           <string>Shadow quality</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbxShadowQuality">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Low</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Medium</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>High</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Low Soft</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Medium Soft</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>High Soft</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="lblPrimitives">
          <property name="text">
           <string>Displayed primitive per point</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbxPrimitives">
          <property name="currentIndex">
           <number>3</number>
          </property>
          <item>
           <property name="text">
            <string>Sphere</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cube</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Minimal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Point</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="lblPointSize">
          <property name="text">
           <string>Point size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spPointSize">
          <property name="minimum">
           <double>0.010000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
          <property name="value">
           <double>0.800000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderPointSize">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>80</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblAxeNameX">
          <property name="text">
           <string>X axe name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtXaxeName">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblAxeNameY">
          <property name="text">
           <string>Y axe name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtYaxeName">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblAxeNameZ">
          <property name="text">
           <string>Z axe name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtZaxeName">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QGroupBox" name="gbxViewSettings">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>View settings</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QPushButton" name="zoomIn">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="plotter_resource.qrc">
                 <normaloff>:/icons/resources/plus_blue.png</normaloff>:/icons/resources/plus_blue.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="zoomOut">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="plotter_resource.qrc">
                 <normaloff>:/icons/resources/minus_blue.png</normaloff>:/icons/resources/minus_blue.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="lblCamPreset">
               <property name="text">
                <string>Camera preset</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbxCameraPreset">
               <item>
                <property name="text">
                 <string>Front</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Left</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Right</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Behind</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnCameraRest">
               <property name="text">
                <string>Reset camera</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="cbxShowGridAndBackGround">
             <property name="text">
              <string>Show grid and background</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QCheckBox" name="cbxUpdateViewForPointCloud">
               <property name="text">
                <string>During receive data over the network update view with frequency</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spbChangeUpdateRate">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="gbxView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Select points to view</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QGroupBox" name="gbxXselect">
           <property name="title">
            <string>X coordinate source</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTableWidget" name="tblXview">
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Graphic Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>X</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Y</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gbxYselect">
           <property name="title">
            <string>Y coordinate source</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QTableWidget" name="tblYview">
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Graphic Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>X</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Y</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gbxZselect">
           <property name="title">
            <string>Z coordinate source</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QTableWidget" name="tblZview">
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Graphic Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>X</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Y</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbxPointActions">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="title">
         <string>Common actions</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QPushButton" name="btnRefreshViews">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Force Refresh views and unselect all</string>
           </property>
           <property name="icon">
            <iconset resource="plotter_resource.qrc">
             <normaloff>:/icons/resources/refresh.png</normaloff>:/icons/resources/refresh.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUpdateView">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Update view</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCyclicAxisChange">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Cyclic Axes Change</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPointColor">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Change Points Color</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbxStats">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Stat</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="lblDisplayedPoints">
           <property name="text">
            <string>Displayed points</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="edtDisplayedPoints">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="plotter_resource.qrc"/>
 </resources>
 <connections/>
</ui>
